Unable to load campaign save

### Environment
49.7
Windows 10
Java Version 11.0.14.1


### Description
Upon trying to load a save game. The save game will not lead and produces an error.
![javaw_XOikwEX1hd](https://user-images.githubusercontent.com/9296835/155033387-5951f8eb-2940-43a3-9efc-06d71bbdbd44.png)

I was able to load a save from earlier, but when I try to save and load my last good save. I receive the error. 

I did notice in the mekhq.log file, this line. "Caused by: org.xml.sax.SAXParseException; lineNumber: 11182; columnNumber: 4; The element type "unitOrder" must be terminated by the matching end-tag "</unitOrder>".

### Files
 Corrupted Save file: [Uber Weapons Unlimited30270326.cpnx.gz](https://github.com/MegaMek/mekhq/files/8112383/Uber.Weapons.Unlimited30270326.cpnx.gz)

Last good save:  [Uber Weapons Unlimited30270319.cpnx.gz](https://github.com/MegaMek/mekhq/files/8112387/Uber.Weapons.Unlimited30270319.cpnx.gz)

[megamek.log](https://github.com/MegaMek/mekhq/files/8112377/megamek.log)
[mekhq.log](https://github.com/MegaMek/mekhq/files/8112378/mekhq.log)

--- a/MekHQ/src/mekhq/campaign/unit/UnitOrder.java
+++ b/MekHQ/src/mekhq/campaign/unit/UnitOrder.java
@@ -321,7 +321,7 @@ public class UnitOrder extends Unit implements IAcquisitionWork {
         pw.println(MekHqXmlUtil.writeEntityToXmlString(getEntity(), indent, getCampaign().getEntities()));
         MekHqXmlUtil.writeSimpleXMLTag(pw, indent, "quantity", quantity);
         MekHqXmlUtil.writeSimpleXMLTag(pw, indent, "daysToWait", daysToWait);
-        MekHqXmlUtil.writeSimpleXMLOpenTag(pw, --indent, "unitOrder");
+        MekHqXmlUtil.writeSimpleXMLCloseTag(pw, --indent, "unitOrder");
     }
 
     public static UnitOrder generateInstanceFromXML(Node wn, Campaign c) {

